frontend/src/index.jade

```diff
@@ -3,14 +3,14 @@ html
     link(rel="stylesheet", href="https://fonts.googleapis.com/css?family=Titillium+Web")
     link(rel="stylesheet", href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.0/normalize.min.css")
     link(rel="stylesheet", href="https://cdnjs.cloudflare.com/ajax/libs/muicss/0.9.39/css/mui.min.css")
-    link(rel="stylesheet", href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/default.min.css")
+    link(rel="stylesheet", href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.2.0/styles/color-brewer.min.css")
     link(rel="stylesheet", href="static/css/style.css")
 
     script(src="https://use.fontawesome.com/releases/v5.0.13/js/all.js", defer=True)
     script(src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.5.16/vue.min.js")
     script(src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.1.5/jszip.min.js")
 
-    script(src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js")
+    script(src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.2.0/highlight.min.js")
```

Comment:
why the switch from 9.12.0 to 9.2.0?

Reply:
There was some support issue. However, its working now